model-optimizer/mo/middle/passes/mean_scale_values.py

```diff
@@ -23,21 +23,21 @@
 def move_scaleshift_to_preprocess_action(graph, match):
     mean_values = {}
     input_op = match['input_op']
-    scale_shift = match['scale_shift']
-    weights = np.squeeze(match['weights'].value)
-    biases = np.squeeze(match['biases'].value)
+    add_op = match['add']
+    mul_op = match['mul']
+    weights = np.squeeze(match['const_mul_output'].value)
```

Comment:
Why do you squeeze these inputs? Now multiplication and addition supports broadcasting so we the inputs may be of different shapes. We cannot blindly squeeze it.